hbase-common/src/main/java/org/apache/hadoop/hbase/io/crypto/Encryption.java

```diff
@@ -127,79 +180,63 @@ public static Cipher getCipher(Configuration conf, String name) {
   }
 
   /**
-   * Return the MD5 digest of the concatenation of the supplied arguments.
+   * Returns the Hash Algorithm defined in the crypto configuration.
    */
-  public static byte[] hash128(String... args) {
-    byte[] result = new byte[16];
+  public static String getConfiguredHashAlgorithm(Configuration conf) {
+    return conf.get(CRYPTO_KEY_HASH_ALGORITHM_CONF_KEY,
```

Comment:
Use config.getTrimmedStrings() instead?

Reply:
nice, I didn't know this function. I changed the code, thanks!